external/os-autoinst-common/tools/update-deps

```diff
@@ -193,7 +193,7 @@ Usage:
     # update cpanfile and dist/rpm/os-autoinst.spec
     $0
     $0 --specfile dist/rpm/os-autoinst.spec
-    $0 --dockerfile container/travis_test/Dockerfile
+    $0 --dockerfile container/ci/Dockerfile
```

Comment:
* This is supposed to be an example, so it doesn't have to match what we are using.
* Since it's in the subrepo, can you do the change there first, and we try if a subrepo pull works this time?

Reply:
sure, done that in https://github.com/os-autoinst/os-autoinst-common/pull/10

Reply:
included a subrepo pull instead of the openQA-local commit

Reply:
for some reason in 5d84f326c3caecdd10550f2b17fd2a201a1d86fd the `parent` commit wasn't updated, and I suspect that is the source of the merge conflict.
Here `parent` is updated, so I hope a normal pull will work next time.
Like I mentioned previously, a `clone -f` is usually the easiest way (just use the command which was originally used) in case of any conflicts.